python_modules/dagster/dagster/_core/definitions/auto_materialize_rule.py

```diff
@@ -297,6 +297,21 @@ def skip_on_required_but_nonexistent_parents() -> "SkipOnRequiredButNonexistentP
         """
         return SkipOnRequiredButNonexistentParentsRule()
 
+    @public
+    @staticmethod
+    def skip_on_backfill_in_progress(
+        skip_all_partitions_of_backfilling_asset: bool = False,
```

Comment:
this is a bit of a mouthful, although it is quite descriptive haha

maybe just `all_partitions` would be good enough, as the docstring can do the heavy lifting on explaining (and most users won't need to set this boolean)

Reply:
good call, I think `all_partitions` is reasonable